### PR TITLE
Store dashborad

### PR DESCRIPTION
**Pull request by RatiMathur**
### [Please link the Jira ticket that this PR addresses here.]
https://trello.com/c/mmnRTmmU/178-fe-dashboard

## Description
Dashboard and welcome page for afroshop stores is created

## To Test
Goto /welcome and /dashboard to check the UI of pages.

## Checklist
- [ ] I have looked over the diffs.
- [ ] There are no errors/failing tests from my changes on this branch.
- [ ] I have requested a review on this PR and re-assigned the Jira ticket.

## Screenshots
[Any screenshots of the changes can go here.]

## Extra Notes
[Please include any extra notes you may have about the changes here.]

### CODE REVIEW

File: src/components/dashboard/dashboard.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+const Dashboard = () => {
+  
+  const options = [
+    { value: 2023, label: 2023 },
+    { value: 2022, label: 2022 },
+    { value: 2021, label: 2021 },
+  ];
+
+  const headers = [
```
Comment (Eshy10) [inline src/components/dashboard/dashboard.js:29]: @RatiMathur we can keep this in the data folder

File: src/components/dashboard/dashboard.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+            <div className="flex gap-4 items-end">
+              <DropdownSelect options={options} placeholder="2023" />
+              <Button variant= "primary" className="w-[109px]">
+                View All Time
+              </Button>
+            </div>
+          </div>
+
+          <div className="flex gap-4">
+            <SummaryCards
```
Comment (Eshy10) [inline src/components/dashboard/dashboard.js:100]: @RatiMathur we can map through this to avoid DRY

File: src/components/dashboard/dashboard.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+                    View All
+                  </Button>                  
+              </div>
+
+              <BaseTable tableHeaders={headers} data={results} className= 'pl-3'/>
+          </div>
+
+            <div className='border-[0.5px] border-solid border-[#B3B3B3] rounded w-[320px] flex flex-col gap-4 p-6 '>
+               <p className='font-semibold text-base'>Weekly Top Customers</p>
+               <CustomerCard customerImage= {CustomerImage1} customerName= "Paityn Dokidis" customerEmail = "paitynd@gmail.com" numberOrders= "54 orders"/>
```
Comment (Eshy10) [inline src/components/dashboard/dashboard.js]: @RatiMathur same as this place

**Comment by Eshy10** [comment]
@RatiMathur please fix your conflicts

**Review by Eshy10 — CHANGES_REQUESTED** [review]
@RatiMathur overview page is what you are working on. it should appear when i click on the overview menu on the sidebar so we are using "/" route.

Screenshot 2023-12-14 at 19 00 27

The dashboard is squashing the sidebar and the content overflows to the right.

Screenshot 2023-12-14 at 19 00 43

**Comment by Eshy10** [comment]
@RatiMathur we can work on other feedbacks and functionality after the provided feedback

**Comment by Eshy10** [comment]
@RatiMathur please fix your conflicts

**Comment by RatiMathur** [comment]
Conflicts resolved
For now User can click on "add my store" button on welcome page to goto dashboard

**Pull request merged**